src/VstsSyncMigrator.Core/Execution/MigrationContext/WorkItemMigrationContext.cs

```diff
@@ -707,5 +714,29 @@ private WorkItemData ReplayRevisions(List<RevisionItem> revisionsToMigrate, Work
 
             return targetWorkItem;
         }
+
+        internal bool SkipRevisionWithInvalidIterationPath(WorkItemData targetWorkItem)
+        {
+            if (_config.SkipRevisionWithInvalidIterationPath)
+            {
+                var workItem = (WorkItem)targetWorkItem.internalObject;
+                var fails = workItem.Validate();
+
+                if (fails.Count > 0)
```

Comment:
also invert this if, this will lead to less nesting since you can then do the for each one level in
`if (failts.Count == 0)
 {
    return false;
}`